desktop/apps/artwork/components/current_auctions/test/current_auctions.js

```diff
@@ -0,0 +1,103 @@
+import 'moment-timezone'
+import CurrentAuctions from '../current_auctions'
+import React from 'react'
+import { expect } from 'chai'
```

Comment:
Is there consensus about `mocha` vs. `chai`? i.e. wondering if it's worth adding a new test framework here given that they seem pretty similar.